README.md

```diff
@@ -12,3 +12,21 @@ $ ./setup.sh
 docker restart raspap
 Web GUI should be accessible on http://localhost by default
 ```
+## Workaround for arm devices
+To use this container on arm devices you have to make cgroups writable:
+```
+docker run --name raspap -it -d --privileged --network=host --cgroupns=host -v /sys/fs/cgroup:/sys/fs/cgroup:rw --cap-add SYS_ADMIN jrcichra/raspap-docker
```

Comment:
The container name here should be changed.

Reply:
To what exactly?

Reply:
`ghcr.io/raspap/raspap-docker:latest` should be the container pulled by users.

Reply:
thanks, fixed in new commit